src/dotnet/CarbonAware/CarbonAwareConstants.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace CarbonAware;
```

Comment:
Eventually I think this should live in the plugins project where the plugin interface lives too, since that's what it's used for